catalog/admin/module/catalog-admin-module-maplayers/src/main/webapp/map-layers/index.js

```diff
@@ -0,0 +1,263 @@
+            if (value === '') {
+              onUpdate('', 'alpha')
+            } else {
+              const n = Number(value)
+              if (!(n < 0 || n > 1)) {
+                onUpdate(n, 'alpha')
+              }
+            }
+          }} />
+      </Flexbox>
+    </Flexbox>
+    <List>
+      <ListItem
+        primaryText='Advanced Configuration'
+        nestedItems={[
+          (options[provider.get('type')] !== undefined)
+            ? <div key='docs' style={{ textAlign: 'center', margin: '0 15px', marginBottom: 20 }}>
+              <FlatButton
+                primary
+                target='_blank'
+                label='openlayers docs'
+                disabled={options[provider.get('type')].help.ol === undefined}
+                href={options[provider.get('type')].help.ol} />
+              <FlatButton
+                primary
+                target='_blank'
+                label='cesium docs'
+                disabled={options[provider.get('type')].help.cesium === undefined}
+                href={options[provider.get('type')].help.cesium} />
+            </div> : null,
+          <div key='ace' style={{ margin: '0 15px' }}>
+            <Error errorText={error.get('buffer')}>
+              <AceEditor
+                mode='json'
+                theme='github'
+                fontSize={15}
+                tabSize={2}
+                width='100%'
+                height='400px'
+                enableBasicAutocompletion
+                name={provider.get('url')}
+                value={provider.get('buffer')}
+                onChange={(value) => onUpdate(value, 'buffer')} />
+            </Error>
+          </div> ]} />
+    </List>
+  </div>
+)
+
+let FixedHeader = ({ muiTheme: { palette }, disabled = true, onSave, onReset }) => (
+  <div style={{
+    position: 'fixed',
+    top: 0,
+    right: 0,
+    left: 0,
+    zIndex: 100,
+    background: palette.backdropColor
+  }}>
+    <Flexbox
+      flex='1'
+      justifyContent='space-between'
+      alignItems='center'
+      style={{
+        maxWidth: 960,
+        margin: '0 auto',
+        padding: '0 20px',
+        borderBottom: `1px solid ${palette.disabledColor}` }}>
+      <Title>Map Layers Configuration</Title>
+      <div>
+        <RaisedButton disabled={disabled} primary label='save' onClick={onSave} style={{ marginRight: 10 }} />
+        <FlatButton disabled={disabled} secondary label='reset' onClick={onReset} />
+      </div>
+    </Flexbox>
+  </div>
+)
+
+FixedHeader = muiThemeable()(FixedHeader)
+
+const MapLayers = ({ onFetch, onUpdate, onSave, onReset, onMessage, disabled, providers = [], errors, loading, message }) => (
+  <Spinner submitting={loading}>
+    <Mount on={onFetch} />
+    <FixedHeader onReset={onReset} onSave={onSave} disabled={disabled} />
+
+    <div style={{ paddingTop: 96 }}>
+      {providers.map((provider, i) =>
+        <Paper key={i} style={{ position: 'relative', marginTop: 20 }}>
+          <IconButton
```

Comment:
I added a reset button in-case someone makes a mistake. I feel like it should be enough.